appveyor.yml

```diff
@@ -5,6 +5,10 @@ environment:
   - JULIAVERSION: "julianightlies/bin/winnt/x86/julia-latest-win32.exe"
   - JULIAVERSION: "julianightlies/bin/winnt/x64/julia-latest-win64.exe"
 
+allow_failures:
```

Comment:
don't think this syntax worked on appveyor, I believe it needs a matrix like on travis